### PR TITLE
Fix shapez exports not being exposed in production

### PR DESCRIPTION
**Pull request by dengr1065**
Initially caused by a mistake in #32. Along with the fix I removed the remaining part of broken export setters.

### CODE REVIEW

File: src/js/mods/modloader.js [hunk 1]
```diff
@@ -104,38 +103,33 @@ export class ModLoader {
...
+            regExp: /\.[jt]sx?$/,
+            exclude: /\.d\.ts$/,
+        });
 
-                    Object.defineProperty(exports, member, {
-                        get() {
-                            return module[member];
-                        },
-                        set(v) {
-                            module.__$S__(member, v);
```
Comment (chunkybanana) [inline src/js/mods/modloader.js:131]: This set method never worked in the first place, right?

**Review by chunkybanana — APPROVED** [review]
lgtm

File: src/js/mods/modloader.js (hunk 1 above)
Comment (dengr1065) [reply src/js/mods/modloader.js:131]: Correct, it *was* working in development builds, but not production (due to optimization apparently). The ES specifications says that imports are "live references" but they're immutable. Note that `shapez.` exports are *not* live references, which is why some bundled mod code may not work the same way as `shapez.`-prefixed code does.

**Comment by cody-ferguson** [comment]
What mistake did I make?

**Pull request merged**